packages/playback-core/src/index.ts

```diff
@@ -260,16 +269,22 @@ export const loadMedia = (
     console.warn("attempting to load media before mediaEl exists");
     return;
   }
-  const { preferMse } = props;
+  const { preferMse, streamType } = props;
   const type = getType(props);
   const hlsType = type === ExtensionMimeTypeMap.M3U8;
 
   const canUseNative = !type || (mediaEl?.canPlayType(type) ?? true);
   const hlsSupported = Hls.isSupported();
+  const userAgentStr = window?.navigator?.userAgent ?? "";
+  // NOTE: Native HLS playback on Android for LL-HLS has been flaky, so we're prefering
+  // MSE for those conditions for now. (CJP)
+  const isAndroid = userAgentStr.toLowerCase().indexOf("android") !== -1;
+  const defaultPreferMse = isAndroid && streamType === StreamTypes.LL_LIVE;
```

Comment:
maybe in the future we could also look into iPadOS>13 MSE performance vs native HLS?

also wondering if there's a difference between Android tablet vs Android mobile but probably not.

Reply:
There shouldn't be really any difference between android tables and android phones, at least, not beyond regular differences in hardware/manufacturer.

Reply:
yeah for now we're not forcing non-native by default unless the native playback is janked-up or doesn't support it (aka android for ll-hls, unless/until they fix it).